samples/rag-cosmosdb/README.md

```diff
@@ -0,0 +1,57 @@
+# Semantic Search Demo
+
+## Prerequisites
+
+The sample is available in the following language stacks:
+
+* [C# on the isolated worker](csharp-ooproc/)
+
+Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
+
+## Running the sample
```

Comment:
Is it possible to run this locally using the Cosmos DB emulator? Or are these features only available in the Azure-hosted Cosmos DB flavors?

Reply:
These features are only available in azure hosted cosmos DB flavors. We would need the emulator to create a MongoDB vcore resource, which I don't think it does today.

Reply:
Yeah I just tried it out with the mongo DB cosmos emulator and the request times out because it expects a MongoDB vcore instance, instead of a regular cosmos DB instance with a mongo db connection string

Reply:
Got it. Thanks for confirming.